src/runtime/node/internal/async_hooks/async-hook.ts

```diff
@@ -22,34 +24,24 @@ class _AsyncHook implements asyncHooks.HookCallbacks {
     return this;
   }
 
-  init(asyncId: number, type: string, triggerAsyncId: number, resource: any) {
-    if (this._enabled && this._callbacks.init) {
-      this._callbacks.init(asyncId, type, triggerAsyncId, resource);
-    }
+  get [Symbol("init")]() {
```

Comment:
Shouldn't that be `Symbol.for()`?

Reply:
I looked into this:
Node uses `Symbol("...")` and has a ref in internal.

I think that having `[Symbol(...)]` makes the code unreachable? Is that what we want here? (maybe to satisfy the type).

Reply:
Yeah same situation as Node (i wish they used `Symbol.for` we have so many issues with this always)

My goal was mainly semantic. it doesn't really matter for private symbols I guess.